packages/syft/src/syft/service/worker/worker_stash.py

```diff
@@ -59,13 +59,13 @@ def get_worker_by_name(
 
     def update_consumer_state(
         self, credentials: SyftVerifyKey, worker_uid: UID, consumer_state: ConsumerState
-    ):
+    ) -> Result[Ok, Err]:
```

Comment:
`Result[T, E]` is an alias for `Union[Ok[T], Err[E]]`. This returns `Union[Ok[str], Err[str]]` which should be `Result[str, str]`.

Reply:
thanks @kiendang. This is very informative